db/schema.rb

```diff
@@ -9,714 +9,696 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended to check this file into your version control system.
+# It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20160519030413) do
+ActiveRecord::Schema.define(version: 20160602153918) do
 
-  create_table "account_users", :force => true do |t|
-    t.integer  "account_id",               :null => false
-    t.integer  "user_id",                  :null => false
-    t.string   "user_role",  :limit => 50, :null => false
-    t.datetime "created_at",               :null => false
-    t.integer  "created_by",               :null => false
+  create_table "account_users", force: :cascade do |t|
+    t.integer  "account_id", limit: 4,  null: false
```

Comment:
These additions of `limit` happen just from a straight schema:dump? Just curious.